src/otx/v2/adapters/torch/mmengine/mmdet/modules/models/backbones/imgclsmob.py

```diff
@@ -0,0 +1,164 @@
+"""Backbone of pytorchcv for mmdetection backbones."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Callable
+
+from mmcv.cnn import build_activation_layer, build_norm_layer
+from mmdet.registry import MODELS
+from mmengine.dist import get_dist_info
+from pytorchcv.model_provider import _models
+from pytorchcv.models.model_store import download_model
+from torch import Tensor, distributed, nn
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from otx.v2.api.utils.logger import get_logger
+
+# ruff: noqa: SLF001
+
+
+def replace_activation(model: nn.Module, activation_cfg: dict) -> nn.Module:
+    """Replace activate funtion."""
+    for name, module in model._modules.items():
+        if len(list(module.children())) > 0:
+            model._modules[name] = replace_activation(module, activation_cfg)
+        if "activ" in name:
+            if activation_cfg["type"] == "torch_swish":
+                model._modules[name] = nn.SiLU()
+            else:
+                model._modules[name] = build_activation_layer(activation_cfg)
+    return model
+
+
+def replace_norm(model: nn.Module, cfg: dict) -> nn.Module:
+    """Replace norm funtion."""
+    for name, module in model._modules.items():
+        if len(list(module.children())) > 0:
+            model._modules[name] = replace_norm(module, cfg)
+        if "bn" in name:
+            model._modules[name] = build_norm_layer(cfg, num_features=module.num_features)[1]
+    return model
+
+
+def multioutput_forward(self: nn.Module, x: Tensor) -> list[Tensor]:
```

Comment:
?
```suggestion
def multi_output_forward(self: nn.Module, x: Tensor) -> list[Tensor]:
```

Reply:
Also here `Tensor` is used to annotate, while the previous modules use `torch.Tensor`. Would be good to stick to one of them for consistency